swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md

```diff
@@ -183,69 +162,69 @@ root, you can remove the path as the follows:
 ```ballerina
 import ballerina/graphql;
 
-service on new graphql:Listener(9000) {
+service on new graphql:Listener(9090) {
```

Comment:
On the previous line `you can remove the path as the follows:` should be changed to `you can remove the path as follows:`